integration_tests/testsuite/deploy_app.py

```diff
@@ -63,14 +70,41 @@ def deploy_app(image, appdir):
         os.chdir(owd)
 
 
+def stop_app():
+    if not DEPLOYED_VERSION:
+        logging.error('App was never deployed!')
+    logging.debug('Stopping application at version %s', DEPLOYED_VERSION)
+    stop_command = ['gcloud', 'app', 'versions', 'stop', DEPLOYED_VERSION]
```

Comment:
Ah cool, thought I had one fail because it wasn't stopped so I added that. Guess that was just a random failure.